solutions/pytorch_inference/Makefile

```diff
@@ -0,0 +1,67 @@
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER    = $(TOP)/scripts/appbuilder
+APP_NAME      = python3
+
+APP_PATH      = /home/user/miniconda/bin/python3
+PACKAGE_PATH  = myst/bin/$(APP_NAME)
+ROOTFS        = ext2rootfs
+TIMEOUT = 180
+
+QUERY_INTERVAL = 2
+SHUTDOWN_WAIT = 2 
+
+all: appdir build
+
+appdir:
+	$(APPBUILDER) -v -d Dockerfile.rootfs
```

Comment:
`-v` generates a lot of docker build log. In pipeline run, might be wasting storage, without much value